package.json

```diff
@@ -10,8 +10,10 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "pnpm": "^8.14.1",
```

Comment:
Why you added `pnpm` to the dependencies?

if you want to use `pnpm` as a package manager you should add as global package.
```
npm install -g pnpm
```

Reply:
It didn't work anyway so I tried both globally and locally. i will remove it

Reply:
What exactly did not work?

Reply:
running scripts is disabled on this system or smth like that

Reply:
Didn't understand, anyways you should set up `pnpm` as global on your machine.

If there is anything, I'm happy to hop on a call and help.

Reply:
Already solved

Reply:
Any comment you did solve, please press the `Resolve conversation` button underneath it.